rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -552,6 +556,133 @@
           }
         }
       }
+    },
+    "url": {
+      "type": "string",
+      "title": "URL",
+      "placeholder": "Example: http://server.mediatheque.ch/CH-000019-X:223094.file",
```

Comment:
This is not useful. No placeholder was added for field `electronicLocator` in the document ([see here](https://github.com/rero/rero-ils/blob/dev/rero_ils/modules/documents/jsonschemas/documents/document_electronic_locator-v0.0.1.json))

```suggestion
```